### PR TITLE
Fix question tree

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?226827

@orangejenny  @gcapalbo

### CODE REVIEW

**Comment by gcapalbo** [comment]
:+1: from me, will give @orangejenny a chance to review as well

File: src/javaRosa/util.js [hunk 1]
```diff
@@ -273,5 +292,6 @@ define([
         insertOutputRef: insertOutputRef,
         looksLikeMarkdown: looksLikeMarkdown,
         parseXLSItext: parseXLSItext,
+        outputToXPathOrHashtag: outputToXPathOrHashtag,
```
Comment (millerdev) [inline src/javaRosa/util.js]: What do you think of exposing two separate functions? It looks like all uses of this have hard-coded arguments. I'm thinking
```js
outputToHashtag: _.partial(outputToXPathOrHashtag, "toHashtag"),
outputToXPath: _.partial(outputToXPathOrHashtag, "toXPath"),
```

Comment (millerdev) [inline src/javaRosa/util.js]: oh, I think no need to use partial, just call it:
```js
outputToHashtag: outputToXPathOrHashtag("toHashtag"),
outputToXPath: outputToXPathOrHashtag("toXPath"),
```

**Comment by orangejenny** [comment]
looks good to me, not merging in case you want to respond to daniels' comment

**Comment by millerdev** [comment]
👍 from me. up to you if you want to implement my suggestion in this PR or another

**Comment by emord** [comment]
Self merging to get it into deploy today. Will implement @millerdev feedback in another PR

**Pull request merged**